### PR TITLE
[GeoMechanicsApplication] Remove duplicated function body and change CalculateIntegrationCoefficient function to use return value

### PR DESCRIPTION
**Pull request by huhaas**
Remove duplicated function body,
Change CalculateIntegrationCoefficient function to use return value.

### CODE REVIEW

File: applications/GeoMechanicsApplication/custom_conditions/U_Pw_face_load_condition.cpp [hunk 1]
```diff
@@ -79,70 +77,60 @@ void UPwFaceLoadCondition<TDim,TNumNodes>::
...
-                                    const double& Weight)
+double UPwFaceLoadCondition<2,3>::
+    CalculateIntegrationCoefficient(const Matrix& Jacobian, const double& Weight)
 {
     double dx_dxi = Jacobian(0,0);
     double dy_dxi = Jacobian(1,0);
 
     double ds = sqrt(dx_dxi*dx_dxi + dy_dxi*dy_dxi);
 
-    rIntegrationCoefficient = ds * Weight;
```
Comment (avdg81) [inline applications/GeoMechanicsApplication/custom_conditions/U_Pw_face_load_condition.cpp:107]: Just for later clean up: this function body is repeated two times (see the function bodies where the template arguments `<2,4>` and `<2,5>` are used). There are some minor differences though in the use of `const`, `sqrt` vs `std::sqrt` and whitespace. Would be nice if we could bring this back to just a single function. But again, not for now, but as follow-up work.

**Review by avdg81 — DISMISSED** [review]
Nice cleanup. Thanks!

File: applications/GeoMechanicsApplication/custom_conditions/U_Pw_normal_lysmer_absorbing_condition.cpp [hunk 2]
```diff
@@ -534,55 +532,21 @@ void UPwLysmerAbsorbingCondition<2, 2>::CalculateRotationMatrix( BoundedMatrix<d
         rRotationMatrix(1, 0) = v_x[1];
         rRotationMatrix(1, 1) = -v_x[0];
     }
+}
 
+template< >
+void UPwLysmerAbsorbingCondition<2, 2>::CalculateRotationMatrix( BoundedMatrix<double, 2, 2>& rRotationMatrix, const Element::GeometryType& rGeom)
+{
+    //Line_2d_2
+    CalculateRotationMatrix2D(rRotationMatrix, rGeom);
```
Comment (aronnoordam) [inline applications/GeoMechanicsApplication/custom_conditions/U_Pw_normal_lysmer_absorbing_condition.cpp]: better naming would be CalculateRotationMatrix2DLine

File: applications/GeoMechanicsApplication/custom_conditions/U_Pw_normal_lysmer_absorbing_condition.cpp [hunk 3]
```diff
@@ -160,13 +159,12 @@ void UPwLysmerAbsorbingCondition<TDim, TNumNodes>::CalculateDampingMatrix(Matrix
         GeoElementUtilities::CalculateNuMatrix<TDim, TNumNodes>(nu_matrix, r_n_container, g_point);
 
         //Compute weighting coefficient for integration
-        this->CalculateIntegrationCoefficient(absorbing_variables.IntegrationCoefficient,
-            jacobians[g_point],
-            r_integration_points[g_point].Weight());
+        double IntegrationCoefficient = this->CalculateIntegrationCoefficient(jacobians[g_point],
```
Comment (aronnoordam) [inline applications/GeoMechanicsApplication/custom_conditions/U_Pw_normal_lysmer_absorbing_condition.cpp]: integration_coefficient,  (to stay consistent with the kratos Styling)

File: applications/GeoMechanicsApplication/custom_conditions/U_Pw_normal_flux_interface_condition.cpp [hunk 4]
```diff
@@ -64,7 +64,6 @@ void UPwNormalFluxInterfaceCondition<TDim,TNumNodes>::
     array_1d<double,TNumNodes> Np;
     array_1d<double,TNumNodes> PVector;
     double NormalFlux;
-    double IntegrationCoefficient;
```
Comment (aronnoordam) [inline applications/GeoMechanicsApplication/custom_conditions/U_Pw_normal_flux_interface_condition.cpp:67]: integration_coefficient, (to stay consistent with the kratos Styling)

**Review by avdg81 — APPROVED** [review]

**Pull request merged**